src/test/java/be/seeseemelk/mockbukkit/entity/BoggedMockTest.java

```diff
@@ -0,0 +1,71 @@
+package be.seeseemelk.mockbukkit.entity;
+
+import be.seeseemelk.mockbukkit.MockBukkit;
+import be.seeseemelk.mockbukkit.ServerMock;
+import net.kyori.adventure.sound.Sound;
+import org.bukkit.entity.EntityType;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class BoggedMockTest
+{
+	private ServerMock server;
+	private BoggedMock bogged;
+
+	@BeforeEach
+	void setUp()
+	{
+		server = MockBukkit.mock();
+		bogged = new BoggedMock(server, UUID.randomUUID());
+	}
+
+	@AfterEach
+	void tearDown()
+	{
+		MockBukkit.unmock();
+	}
+
+	@Test
+	void shear_GivenShearSequence()
```

Comment:
Probably also make sense to check if the sound of sheering has been heard

Reply:
I did not include it, because i'm using the setUp and tearDown metod to mock and unmock.